src/components/NoteLabelItem/NoteLabelItem.css

```diff
@@ -0,0 +1,64 @@
+.note__label_display {
+	display: flex;
+	gap: 5px;
+	padding: 6px;
+	flex-flow: row wrap;
+}
+
+.note--displayed .note_listing__label {
+	display: flex;
+	flex-flow: row wrap;
+	padding: 2px;
+	gap: 4px;
+	padding-top: 5px;
+}
+
+.label__li {
+	position: relative;
+	border: 1px solid var(--COLOR-TXT);
+	border-radius: 40/80%;
+	padding: 4px 8px;
+	border-radius: 15px;
+	list-style-type: none;
+	display: flex;
+	align-items: center;
+	gap: 5px;
+	cursor: default;
+	font-size: 15px;
+}
+
+.note--displayed .label__li {
+}
+
+.label__li:hover {
+}
+
+.label__item {
+}
```

Comment:
this className is not in use Ig, you can remove them

Reply:
will update it